tensorflow/core/ops/math_ops.cc

```diff
@@ -433,6 +433,37 @@ Note, above `Q(a, x)` (`Igammac`) is the upper regularized complete
 Gamma function.
 )doc");
 
+REGISTER_OP("Zeta")
+    .Input("x: T")
+    .Input("q: T")
+    .Output("z: T")
+    .Attr("T: {float, double}")
+    .Doc(R"doc(
+Compute the Hurwitz zeta function `\zeta(x, q)`.
```

Comment:
Just realised that you use \\( math \\) for inline math. Shall I fix all the other documentation characters in the process? E.g. see line [122](https://github.com/tillahoffmann/tensorflow/blob/master/tensorflow/core/ops/math_ops.cc#L122) in the same file.